content/300-guides/200-deployment/201-serverless/400-deploy-to-aws-lambda.mdx

```diff
@@ -1,23 +1,33 @@
 ---
-title: 'Deploy to AWS Lambda with Serverless Framework'
-metaTitle: 'Deploy your Serverless Framework project with Prisma'
-metaDescription: 'Learn how to deploy your Prisma-backed applications to AWS Lambda with Serverless Framework'
+title: 'Deploy to AWS Lambda'
+metaTitle: 'Deploy your Prisma-backed project to AWS Lambda'
```

Comment:
I'm not quite comfortable with the wording used here 😅

The wording for the suggestion is also not quite there but we can let this simmer for now.
```suggestion
metaTitle: 'Deploy your application using Prisma to AWS Lambda'
```

Reply:
"deploy ... using prisma"

I think "Prisma-backed" was a nice solution for that problem :D